### DISCUSSION TITLE
[Question]:

### DISCUSSION THREAD

**Discussion by Matthew-Modvans** [carbon-design-system/carbon-components-svelte#1725] [Q&A]
### Question for Carbon

Hello, I'm trying to implement carbon components into a svelte web app I'm working on. I installed the library using npm, and in the script tag, I have these two lines

	import "carbon-components-svelte/css/g80.css";
	import { Button } from 'carbon-components-svelte';

But whenever I try to create the button element, I just get the error 'Button' is not defined. Consider adding a <script> block with 'export let Button' to declare a propsvelte(missing-declaration). Any help would be great, thanks!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)

**Comment by theetrain** [comment]
Hi @Matthew-Modvans, could you share your .svelte file here? That way I can have more context of the issue.

Typically you would use the button and CSS code in a .svelte file like this:

```html
<script>
  import "carbon-components-svelte/css/g80.css";
  import { Button } from 'carbon-components-svelte';
</script>

<Button>Click here</Button>
```
